pymeasure/experiment/results.py

```diff
@@ -33,11 +33,13 @@
 from string import Formatter
 
 import pandas as pd
+import pint
 
 from .procedure import Procedure, UnknownProcedure
 
 log = logging.getLogger(__name__)
 log.addHandler(logging.NullHandler())
+u = pint.get_application_registry()
```

Comment:
I think the canonical way is to call this `ureg`.

Reply:
I took the inspiration from instrumentkit (the only pint-using project I know). I guess the reasoning was, that u is far shorter and u.m as unit of meters is more concise than ureg.m.
I'm fine with `ureg` as well.